src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -7,6 +7,95 @@
+        public void Add(short jsonValue) { }
+        public void Add(int jsonValue) { }
+        public void Add(long jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(sbyte jsonValue) { }
+        public void Add(float jsonValue) { }
+        public void Add(string jsonValue) { }
+        public void Add(System.Text.Json.JsonNode jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(ushort jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(uint jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(ulong jsonValue) { }
+        public void Clear() { }
+        public bool Contains(bool jsonValue) { throw null; }
+        public bool Contains(byte jsonValue) { throw null; }
+        public bool Contains(double jsonValue) { throw null; }
+        public bool Contains(short jsonValue) { throw null; }
+        public bool Contains(int jsonValue) { throw null; }
+        public bool Contains(long jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(sbyte jsonValue) { throw null; }
+        public bool Contains(float jsonValue) { throw null; }
+        public bool Contains(string jsonValue) { throw null; }
+        public bool Contains(System.Text.Json.JsonNode jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(ushort jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(uint jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(ulong jsonValue) { throw null; }
+        public void CopyTo(System.Text.Json.JsonNode[] array, int arrayIndex) { }
+        public System.Collections.Generic.IEnumerator<System.Text.Json.JsonNode> GetEnumerator() { throw null; }
+        public int IndexOf(System.Text.Json.JsonNode item) { throw null; }
+        public void Insert(int index, bool item) { }
+        public void Insert(int index, byte item) { }
+        public void Insert(int index, double item) { }
+        public void Insert(int index, short item) { }
+        public void Insert(int index, int item) { }
+        public void Insert(int index, long item) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Insert(int index, sbyte item) { }
+        public void Insert(int index, float item) { }
+        public void Insert(int index, string item) { }
+        public void Insert(int index, System.Text.Json.JsonNode item) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Insert(int index, ushort item) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Insert(int index, uint item) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Insert(int index, ulong item) { }
+        public bool Remove(System.Text.Json.JsonNode item) { throw null; }
```

Comment:
What happens with duplicate items? Remove the first? or all instances?

Reply:
Checked this in documentation, it also says it should remove the first occurrence. But we can think of adding `RemoveAll`?

Reply:
mm probably it wouldn't be very useful, I think I'm fine with just having the default list functionality.